server/raft.go

```diff
@@ -2564,7 +2564,7 @@ func (n *raft) processAppendEntry(ae *appendEntry, sub *subscription) {
 			select {
 			case n.applyc <- &CommittedEntry{n.commit, ae.entries[:1]}:
 			default:
-				n.debug("Failed to place snapshot entry onto our apply channel")
+				n.warn("Failed to place snapshot entry onto our apply channel")
```

Comment:
If i saw this, I would have no way to figure out what I could do about that. Is it just a wait and see kind of situation or what would be the remedy? Warn level logs need to be actionable some how - or indicate that they will hopefully clear

Reply:
I promoted it so it could be more easily reported to us via support channels etc.